gbpservice/neutron/services/servicechain/plugins/ncp/node_drivers/nfp_node_driver.py

```diff
@@ -273,6 +274,16 @@ def get_plumbing_info(self, context):
 
         if service_type in [pconst.FIREWALL, pconst.VPN]:
             plumbing_request['plumbing_type'] = 'gateway'
+            # plumber will return stitching network PT instead of consumer
+            # as chain is instantiated while creating provider group.
+            if (self._check_for_fw_vpn_sharing(context, service_type) and
```

Comment:
Will the node create order always FW and followed by VPN? Is this the reason why we always skip interface create for VPN?